src/assets/illustrations/empty-snapshot.svg

```diff
@@ -0,0 +1,25 @@
+<svg width="181" height="104" viewBox="0 0 181 104" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I dont think we need to add this svg here. There's the `FallbackIcon` component already that can be used